runtime/j9vm/exports.cmake

```diff
@@ -377,6 +377,11 @@ if(NOT JAVA_SPEC_VERSION LESS 17)
 		JVM_DumpClassListToFile
 		JVM_DumpDynamicArchive
 	)
+	if(J9VM_ZOS_3164_INTEROPERABILITY)
+		jvm_add_exports(jvm
```

Comment:
We've been keeping uma alive, although I don't think we still have a reason to do so. I'm fine with not updating the uma exports.

Reply:
oh I missed it, I see you updated uma exports anyway.